sale_advance_payment/views/sale_view.xml

```diff
@@ -11,7 +11,7 @@
                     string="Pay sale advanced"
                     type="action"
                     groups="account.group_account_invoice"
-                    attrs="{'invisible': [('state', 'in', ['done','cancel'])]}"
+                    invisible="state == 'cancel'"
```

Comment:
We don't want it anymore to be invisible when the state is done ? It does not look like a migration action